features/media-queries-overflow.yml

```
@@ -0,0 +1,9 @@
+name: Overflow media features
+description: The `overflow-block` and `overflow-inline` CSS media features set styles based on how a user's device will show content that is larger than the screen. Options are `none` (for instance, a billboard), `scroll` (for instance, a laptop), or `paged` (for instance, a printer).
```

```suggestion
description: The `overflow-block` and `overflow-inline` CSS media features set styles based on the way a device displays content that's larger than its container element. For example, a laptop lets users scroll to reveal content, while a printer displays the overflowing content on additional pages.
```

Since this can be used for printers, I don't think saying user's device helps here. Also removing "screen" for the same reason.

I'm not sure about the second sentence with the different values. The billboard example confuses me a bit. When would that happen? So I tried to rephrase above.